pkg/minikube/openshiftversions/openshift_versions_test.go

```diff
@@ -112,7 +112,7 @@ func TestGetVersionsMalformed(t *testing.T) {
 
 	_, err := getVersions()
 	if err == nil {
-		t.Fatal("Malformed version value was returned from URL but no error was thrown")
+		t.Fatal("Incorrect version value returned from URL but no error reported.")
```

Comment:
Well I don't think that "malformed URL" is a good phrase, and when I look it up I see tht it means basically the same thing. I'll choose my battles though, if you insist on this I'll go with what you say.